db/migrate/20210228033155_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :body
+      t.string :slug
+      t.text :preview
+
+      t.timestamps
+    end
+    add_index :posts, :slug, unique: true
```

Comment:
This and other posts files were generated with `rails g model Post title:string body:text slug:uniq preview:text`.

I want slugs to be the identifier in the url instead of id so I decided they should be unique strings. Generating them with `slug:uniq` added the `add_index :posts, :slug, unique: true` too for faster lookups